xmlfmt_test.go

```diff
@@ -18,56 +16,28 @@ const xml4 = `<?xml version="1.0" encoding="UTF-8">
   <Relationships xmlns="http://schemas.openxmlformats.org/package/2006/relationships"><Relationship Id="rId1" Type="http://schemas.openxmlformats.org/package/2006/relationships/metadata/core-properties" Target="docProps/core.xml"/><Relationship Id="rId2" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/extended-properties" Target="docProps/app.xml"/><Relationship Id="rId3" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/officeDocument" Target="ppt/presentation.xml"/></Relationships>`
 
 func Example_output() {
-	x3 := xmlfmt.FormatXML(xml3, "\t", " ")
-	x2 := xmlfmt.FormatXML(xml2, "x ", " ")
+	x3 := FormatXML(xml3, "\t", " ")
+	x2 := FormatXML(xml2, "x ", " ")
 	_ = x2
 	_ = x3
-	x1 := xmlfmt.FormatXML(xml1, "", "  ")
+	x1 := FormatXML(xml1, "", "  ")
 	fmt.Println(x1)
-	// Output:
```

Comment:
Please don't make unnecessary changes like this. 

When the new change has broken things, hiding it away is not the appropriate way of doing things.

Reply:
It has been restored